spk/sickbeard-custom/Makefile

```diff
@@ -36,6 +37,8 @@ include ../../mk/spksrc.spk.mk
 
 .PHONY: sickbeard-custom_extra_install
 sickbeard-custom_extra_install:
+	install -m 755 -d $(STAGING_DIR)/etc
+	cp src/sickbeard-custom.sc $(STAGING_DIR)/etc/sickbeard-custom.sc
```

Comment:
Can't you tell the resource file to grab it from the scripts dir, where the file is currently located? Otherwise you're including the same file twice in the package.